pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -58,12 +62,35 @@ type DbSpec struct {
 }
 
 type ApiSpec struct {
-	HubConfigUrl           string `json:"hubConfigUrl,omitempty"`
```

Comment:
is this field removed?

Reply:
yes, as we have configs in configmap now

Reply:
is it backward comapatible? will the upgrade be smooth?

Reply:
Upgrade is smooth, but we'll have to keep the `HubConfigUrl` field in the struct 
In the release notes we'll have to add that we are deprecating this field and in the next release we'll remove it

Regarding backward compatibility if users have already used their config.yaml and provided their data then in
that case user's data will be lost, we'll have to document this as well in the release notes